tests/test_slots.py

```diff
@@ -0,0 +1,26 @@
+from falcon import Request, Response
+import falcon.testing as testing
+
+
+class TestSlots(testing.TestBase):
+
+    def __init__(self):
```

Comment:
If `self.failed` is not necessary (see below), the `__init__()` method can be removed.